<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Einstellungen</class>
 <widget class="QDialog" name="Einstellungen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Einstellungen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/icons/res/icons/settings_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/res/icons/settings_FILL0_wght400_GRAD0_opsz24.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Einstellungen</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../ressources.qrc">:/icons/res/icons/settings_FILL0_wght400_GRAD0_opsz24.svg</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Allgemein</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Stil (Neustart erforderlich):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmb_stil">
          <item>
           <property name="text">
            <string>Vocitrainer</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Windows</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QCheckBox" name="box_willkommenMeldung">
          <property name="text">
           <string>Willkommen-Meldung zum Start anzeigen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmd_anzeigen">
          <property name="text">
           <string>Anzeigen</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>38</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Standartsprache für neues Set:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmb_sprache">
          <item>
           <property name="text">
            <string>Englisch</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Französisch</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Latein</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Spanisch</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Italienisch</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../ressources.qrc">:/icons/res/icons/note_stack_FILL0_wght400_GRAD0_opsz24.svg</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Intelligenter Lernmodus (&quot;Set lernen&quot;)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Millersche Zahl</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmb_mz">
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7 - Normal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Fehlertoleranz:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmb_ft">
          <item>
           <property name="text">
            <string>Tief</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Normal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hoch</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="box_definitionLernen">
        <property name="text">
         <string>Definition lernen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../ressources.qrc">:/icons/res/icons/manage_accounts_FILL0_wght400_GRAD0_opsz24.svg</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Profil (Marketplace)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="lbl_profilStatus">
          <property name="text">
           <string>Angemeldet / Abgemeldet</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmd_xxMelden">
          <property name="text">
           <string>Abmelden</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lbl_email">
        <property name="text">
         <string>E-Mail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_benutzername">
        <property name="text">
         <string>Benutzername</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_passwort">
        <property name="text">
         <string>Passwort</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>205</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmd_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmd_abbrechen">
       <property name="text">
        <string>Abbrechen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmd_uebernehmen">
       <property name="text">
        <string>Übernehmen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
